include/files.cpp

```diff
@@ -66,9 +66,15 @@ void readFile(std::string fName) {
     int curLineNum = 1;
     while (std::getline(fp, curLine))
     {
-        assembleLine(curLine, curLineNum, romName);
+        if (assembleLine(curLine, curLineNum, romName) == 1) {
+            return;
```

Comment:
This section looks odd. Maybe rethink separation of concern. (Would address this via new issues & new PR if I was you)